### ISSUE TITLE
list in aside (incompatible change)

### ISSUE THREAD

**Issue by mnot** [rfc-format/draft-iab-xml2rfc-v3-bis#167]
In the 2.32 revision of the RNC, the `list` element was removed from `aside`. While 7991bis says that `list` is deprecated, making documents containing it invalid isn't really deprecation -- making this a backwards-incompatible change. This needs to be discussed and documented.

**Comment by reschke** [comment]
FWIW, it's not backwards incompatible as \<aside> did not exist before.

**Comment by mnot** [comment]
`aside` is in 7991.

**Comment by reschke** [comment]
Yes, but AFAIU, backwards compat to 7991 was not considered as relevant.

RFC 7991 was published as snapshot of the then-current draft for the sake of getting implementation experience. It should have got review, discussion and revision years ago.

**Comment by mnot** [comment]
If it isn't in use in any current RFCs or RFCs-to-be, that seems reasonable. I'd still expect this to be listed in a "Changes from RFC7991" section, though.

**Comment by stpeter** [comment]
> If it isn't in use in any current RFCs or RFCs-to-be, that seems reasonable.

By "it" do you mean list in aside, or aside itself? I suppose it might not matter for our purposes here...

> I'd still expect this to be listed in a "Changes from RFC7991" section, though.

Makes sense to me.

**Comment by jrlevine** [comment]
Was a mistake in 7991, <list> was deprecated.

**Comment by stpeter** [comment]
@jrlevine It seems we might want to report an erratum against 7991 for this, too.

**Comment by jrlevine** [comment]
@stpeter did it, erratum #6424

**Comment by jrlevine** [comment]
Paul H points out that <list> is deprecated, and I see it does not occur in any published RFC.
Perhaps we should take it out of <t> also.

**Comment by reschke** [comment]
The reason is that RFC 7791 was supposed to be backwards compatible to RFC 7749. That is, v3 was supposed to deprecrate thinsg for removal in v4.

list inside aside was a mistake; there never was an intent to allow deprecated elements inside new elements.

**Comment by stpeter** [comment]
@jrlevine +1
